woocommerce/woocommerce-gateway-stripe
# Move Alipay and Multibanco inside Stripe settings when UPE is disabled

Fixes #2852 

## Changes proposed in this Pull Request:
- Alipay and Multibanco are also moved inside the Stripe settings page for UPE disabled state.

**Before**
![ali-multi before](https://github.com/woocommerce/woocommerce-gateway-stripe/assets/33387139/ebc5c91c-7385-4447-9f14-82711f9806d3)


**After**
![ali-multi after](https://github.com/woocommerce/woocommerce-gateway-stripe/assets/33387139/c247cc63-6014-49bf-925b-30da6ddf0453)



## Testing instructions
- Go to the Stripe settings page and disable UPE.
- Go to the `Payment methods` tab on the Stripe settings page. Alipay and Multibanco should be listed here.
- Enable/Disable these 2 methods and make sure they are available on the checkout page as per settings.
